package.json

```diff
@@ -121,6 +122,7 @@
     "babel-loader": "^9.1.2",
     "cookie-parser": "^1.4.5",
     "cypress": "^12.0.0",
+    "cypress-real-events": "^1.8.1",
```

Comment:
I couldn't make `@4tw/cypress-drag-drop` library or cypress's `trigger` to work with our block connections.
Possibly, because the `<plain-draggable>` inside of the block exits is conditionally rendered on mouse hover.
And we can't simulate hovering with `@4tw/cypress-drag-drop` or via cypress's `trigger('mouseover')`.
I might be wrong though.

Reply:
Unfortunately, `cypress-real-events` works only in Chromium-based browsers.
This means we won't be able to run cypress tests in Firefox, etc.
Please confirm that it's okay.

Reply:
No concerns about limiting with Chromium-based browsers only. We just need to inform QA about this.